lib/logstash/filters/dissect.rb

```diff
@@ -4,43 +4,145 @@
 
 require "java"
 require "jars/jruby-dissect-library.jar"
-require "jruby_dissect"
+require "jruby_dissector"
 
-# This example filter will replace the contents of the default
-# message field with whatever you specify in the configuration.
+# De-structures text
 #
-# It is only intended to be used as an example.
-class LogStash::Filters::Dissect < LogStash::Filters::Base
+# The dissect filter is a kind of split operation.
```

Comment:
We need more documentation. Why would you use dissect vs grok? Whats the rationale here? Also, please use asciidoc